erpnext/accounts/report/balance_sheet/balance_sheet.py

```diff
@@ -8,11 +8,14 @@
 from erpnext.accounts.report.financial_statements import (get_period_list, get_columns, get_data)
 
 def execute(filters=None):
-	period_list = get_period_list(filters.from_fiscal_year, filters.to_fiscal_year, filters.periodicity, filters.company)
+	period_list = get_period_list(filters.from_fiscal_year, filters.to_fiscal_year, filters.periodicity, filters.accumulated_values)
```

Comment:
Why removed filters.company? Is it very important to have company to get correct fiscal year based on a date.